testing/cover/cover.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"sort"
+
+	"golang.org/x/tools/cover"
+)
+
+func merge(p1, p2 *cover.Profile) *cover.Profile {
```

Comment:
None of the go coverage code I've read would indicate that, but I'm not 100% sure TBH.